apps/datafeeder/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,12 @@
 import { NgModule } from '@angular/core'
 import { Routes, RouterModule } from '@angular/router'
+import { UploadDataComponent } from './components/upload-data/upload-data.component'
 
-const routes: Routes = []
+const routes: Routes = [
+  { path: '', redirectTo: 'georchestra/import', pathMatch: 'full' },
```

Comment:
`georchestra` is the hostname and `import` the base path of the application.
At application level, it should just be `/`